cime_config/acme/allactive/config_pes.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 
-<config_pes>
+<config_pes version="2.0">
```

Comment:
The file will still work without this change but it won't be checked against the schema.